static/index.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Themeroller</title>
```

Comment:
@arthurvr, it was a minor thing, and since the page was going to be replaced, I decided it was okay if we just did the PR.